dashboard/lib/sample_data.rb

```diff
@@ -0,0 +1,181 @@
+include FactoryGirl::Syntax::Methods
+
+class SampleData
+  TEST_ACCOUNT_PASSWORD = "00secret".freeze
+  @@prng = nil
+
+  # Returns a seeded random number generator for consistent test data
+  def self.prng
+    @@prng ||= Random.new(0)
+    @@prng
+  end
+
+  # Creates sample data
+  def self.seed
+    raise "Should not be run outside of development" unless CDO.rack_env?(:development)
+
+    # Create a test teacher
+    teacher = create_teacher "testteacher@code.org", "TestTeacher Codeberg"
+
+    # Create normal-sized and large sections for each of CSF, CSD and CSP
+
+    create_section({teacher: teacher, name: "CSF 1",
+      login_type: Section::LOGIN_TYPE_PICTURE, grade: 2, age_min: 7,
+      age_max_inclusive: 9, script_name: "coursea-2018", num_students: 30,
+      use_imperfect_results: true}
+      )
+
+    create_section({teacher: teacher, name: "CSF 2 (Large)",
+      login_type: Section::LOGIN_TYPE_PICTURE, grade: 2, age_min: 7,
+      age_max_inclusive: 9, script_name: "coursea-2018", num_students: 150,
+      use_imperfect_results: true}
+      )
+
+    create_section({teacher: teacher, name: "CSD 1",
+      login_type: Section::LOGIN_TYPE_EMAIL, grade: 7, age_min: 13,
+      age_max_inclusive: 14, script_name: "csd1-2018", num_students: 30,
+      use_imperfect_results: false}
+      )
+
+    create_section ({teacher: teacher, name: "CSD 2 (Large)",
+      login_type: Section::LOGIN_TYPE_EMAIL, grade: 7, age_min: 13,
+      age_max_inclusive: 14, script_name: "csd1-2018", num_students: 150,
+      use_imperfect_results: false}
+      )
+
+    create_section ({teacher: teacher, name: "CSP 1",
+      login_type: Section::LOGIN_TYPE_EMAIL, grade: 10, age_min: 14,
+      age_max_inclusive: 16, script_name: "csp1-2018", num_students: 30,
+      use_imperfect_results: false}
+      )
+
+    create_section ({teacher: teacher, name: "CSP 2 (Large)",
+      login_type: Section::LOGIN_TYPE_EMAIL, grade: 10, age_min: 14,
+      age_max_inclusive: 16, script_name: "csp1-2018", num_students: 150,
+      use_imperfect_results: false}
+      )
+  end
+
+  # will delete the teacher and all of the teacher's sections and students
```

Comment:
Nit: comment style isn't consistent with the others:
`Delete the teacher and all of the teacher's sections...`